index.d.ts

```diff
@@ -610,6 +622,11 @@ declare module '@primer/components/lib/Details' {
   export default Details
 }
 
+declare module '@primer/components/lib/useDetails' {
```

Comment:
Does this need to be 
```suggestion
declare module '@primer/components/lib/hooks/useDetails' {
```
?

Reply:
It looks like it works as is, so maybe it's fine? Not sure though.

Reply:
🤔 I'm not sure why it worked as is, but it works with either path so I changed it!